index.ts

```diff
@@ -843,43 +844,39 @@ export class CapabilitiesController extends BaseController<any, any> implements
       return end(res.error);
     }
 
-    if (!metadata.id) {
-      metadata.id = uuid();
-    }
+    const id = typeof req.id === 'number' || req.id
```

Comment:
Note: We'll have to remember to update the permissions controller to account for this as well; there is a spot where the `id` is passed in via the metadata rather than the request.